src/igvfd/schemas/curated_set.json

```diff
@@ -69,6 +69,17 @@
                 "transcriptome",
                 "variants"
             ],
+            "enum_descriptions": {
+                "barcodes": "Barcodes are lists of barcodes.",
```

Comment:
... lists of barcodes identifying cells in a single cell experiment.

Reply:
fixed